src/data/javascript-quiz.ts

```diff
@@ -1158,7 +1158,25 @@ const javascriptQuiz = [
     Explanation:
       ".push() returns the new length of the array, after the element has been pushed",
     Link: "https://www.freecodecamp.org/news/how-to-insert-an-element-into-an-array-in-javascript/"
-  }
+  },
+  {
+    Question: "In JavaScript, from following what isn't use to declare a variable?",
+    Answer: "number",
+    Distractor1: "let",
+    Distractor2: "var",
+    Distractor3: "const",
+    Explanation: "JavaScript has three kinds of variable declarations. var declare a variable, optionally initializing it to a value, let declare a block scoped, local variable,optionally initializing it to a value and const declare read-only block scoped constant",
```

Comment:
Let's trim down this quiz question a bit. If users want to learn more they can click on the learn more link to read the article. 

```suggestion
    Explanation: "JavaScript has three kinds of variable declarations: var, let and const.",
```